nlp_architect/utils/embedding.py

```diff
@@ -100,13 +100,15 @@ def get_embedding_matrix(embeddings: dict, vocab: Vocabulary,
     else:
         mat = np.zeros((len(vocab), emb_size))
     for word, wid in vocab.vocab.items():
-        vec = embeddings.get(word.lower(), None)
+        vec = embeddings.get(word, None)
+        if vec is None:
+            vec = embeddings.get(word.lower(), None)
```

Comment:
some word embedding models are lower case only
this method should be modified to support selection of lower/mixed mode

Reply:
done